static_code_analysis.txt

```diff
@@ -32,7 +32,7 @@ Test results:
 156	        assert issubclass(type(synthesizer), BaselineSynthesizer), (
 157	            '`synthesizer` must be an instance of a synthesizer class.'
 158	        )
-159	
+159
```

Comment:
does this file need to be excluded from the linter?

Reply:
Yep, fixed:
https://github.com/sdv-dev/SDGym/pull/402/commits/98dc8bf052b77690b5595381480ada1f7c9f3fe7#diff-50c86b7ed8ac2cf95bd48334961bf0530cdc77b5a56f852c5c61b89d735fd711R199